sql-plugin/src/main/scala/org/apache/spark/sql/rapids/collectionOperations.scala

```diff
@@ -809,28 +813,65 @@ case class GpuArraysZip(children: Seq[Expression]) extends GpuExpression with Sh
 
   private def generateSeqIndices(maxArraySize: ColumnVector): ColumnVector = {
     withResource(GpuScalar.from(0, IntegerType)) { s =>
-      val zeroCV = GpuColumnVector.from(s, maxArraySize.getRowCount.toInt, IntegerType)
-      withResource(zeroCV.getBase) { zero =>
+      withResource(cudf.ColumnVector.fromScalar(s, maxArraySize.getRowCount.toInt)) { zero =>
```

Comment:
Nit: cudf's ColumnVector is already imported and is being used directly one line below.
```suggestion
      withResource(ColumnVector.fromScalar(s, maxArraySize.getRowCount.toInt)) { zero =>
```